tutorials/grails3-send-push-notification-using-firebase-cloud-messaging

```diff
@@ -0,0 +1,196 @@
+# Grails 3 Sending Push Notification using Google Firebase Cloud Messaging
```

Comment:
You'll need to add a metadata header, see any of the other tutorials for examples.